sycl/test/check_device_code/vector/vector_math_ops.cpp

```diff
@@ -25,6 +26,15 @@ using namespace sycl;
 // CHECK-NEXT:    [[ADD_I:%.*]] = add <2 x i32> [[TMP0]], [[TMP1]]
 // CHECK-NEXT:    store <2 x i32> [[ADD_I]], ptr addrspace(4) [[AGG_RESULT]], align 8, {{.*}}
 // CHECK-NEXT:    ret void
+// CHECK-ARR-STORAGE-LABEL: define dso_local spir_func void @_Z7TestAddN4sycl3_V13vecIiLi2EEES2_(
+// CHECK-ARR-STORAGE-SAME: ptr addrspace(4) dead_on_unwind noalias nocapture writable writeonly sret(%"class.sycl::_V1::vec") align 8 [[AGG_RESULT:%.*]], ptr nocapture noundef readonly byval(%"class.sycl::_V1::vec") align 8 [[A:%.*]], ptr nocapture noundef readonly byval(%"class.sycl::_V1::vec") align 8 [[B:%.*]]) {{.*}} {
```

Comment:
Different alignment of load when loading from ext_vector_type and std::array.

Reply:
I think that's fine. SPEC just requires sycl::vec to have same alignment as `sizeof(T) * NumElements`, except for `NumElements=3`.  The alignment of the return value suggests that this is the case.